dev/tests/bundle/tasks.js

```diff
@@ -0,0 +1,63 @@
+/**
```

Comment:
Unfortunately unit tests not cover whole code, because testing gulp is really difficult :/